cylc/flow/scheduler.py

```diff
@@ -1232,10 +1232,18 @@ def _load_workflow_params(self, row_idx, row):
             LOG.info(f"+ cycle point time zone = {value}")
 
     def _load_template_vars(self, _, row):
-        """Load workflow start up template variables."""
-        key, value = row
+        """Load workflow start up template variables.
+
+        If a type has been stored in the database, save this too.
+        """
+        key, value, keytype = row
         # Command line argument takes precedence
         if key not in self.template_vars:
+            # Recast database items according to type.
```

Comment:
@oliver-sanders  - From comments at conference I get the impression that you think that I should be passing these to `cylc.flow.templatevars.eval_vars`?